.circleci/config.yml

```diff
@@ -138,7 +138,10 @@ jobs:
       - store_artifacts:
           path: test-reports
 
-  staging_deploy:
+  multideploy:
```

Comment:
I would prefer this to just be `deploy`. What's the rationale for `multideploy`? I know that's what we call having multiple UAT environments, but if this step is used for that as well as for the single-deploy environments (staging and prod) then I think it gets confusing.

Reply:
Updated.

I actually was using `deploy` but renamed it to `multideploy` to match what was used in ticket